modules/terraform/output.go

```diff
@@ -49,6 +50,134 @@ func OutputRequiredE(t testing.TestingT, options *Options, key string) (string,
 	return out, nil
 }
 
+// parseListOfMaps takes a list of maps and parses the types.
+// It is mainly a wrapper for parseMap to support lists.
+func parseListOfMaps(l []interface{}) ([]map[string]interface{}, error) {
+	var result []map[string]interface{}
+
+	for _, v := range l {
+		m, err := parseMap(v.(map[string]interface{}))
```

Comment:
This will panic if `v` is not a `map[string]interface{}`. We do NOT want our tests to panic, ever, if we can avoid it, so please break the inner part out into a separate line:

```go
asMap, isMap = v.(map[string]interface{})
if !isMap {
  // return error
}

m, err := parseMap(asMap)
```

Reply:
This issue still needs to be resolved.